### PR TITLE
All changes

### PR DESCRIPTION
**Pull request by 0mbi**
- Alter model:
  * gender now with one char and defined choices; to work well with django admin
  * add additional maybe useful flat information
- Write simple model tests for Refugee and Flat
- Add simple logic for fits into flat

### CODE REVIEW

File: refugeerouter/models.py [hunk 1]
```diff
@@ -19,15 +19,27 @@ def __str__(self):
 
 
 class Refugee(models.Model):
+    GENDER_FEMALE='F'
+    GENDER_MALE='M'
+    GENDER_DIVERSE='D'
+    GENDER_CHOICES = [
```
Comment (Lassulus) [inline refugeerouter/models.py:26]: there is probably also unknown (which should be the default IMHO)

Comment (0mbi) [reply refugeerouter/models.py:26]: fixed

**Comment by Lassulus** [comment]
should we squash the migrations?

**Comment by 0mbi** [comment]
I would recommend to bring this to a releasable level and then remove all migrations. 

But yeah squash if you like to. I don't mind any pre-prod commit structure.

**Comment by Lassulus** [comment]
whats missing to get it to releaseable level? I guess squashing and then merging should be fine

**Comment by 0mbi** [comment]
> whats missing to get it to releaseable level? I guess squashing and then merging should be fine

Well a working piece of software that is usable? However, squashed it. Let's go on

**Comment by Lassulus** [comment]
I meant squashing the migrations, not the code, but nvm :D

File: refugeerouter/models.py [hunk 2]
```diff
@@ -19,15 +19,29 @@ def __str__(self):
...
+    gender = models.CharField(
+        max_length=1,
+        choices=GENDER_CHOICES,
+        default=GENDER_UNKNOWN
+    )
     contact_data = models.CharField(max_length=1024)
     origin = models.CharField(max_length=1024)
     origin_checked = models.BooleanField(default=False)
-    group = models.ForeignKey(Group, on_delete=models.CASCADE)
+    group = models.ForeignKey(Group, on_delete=models.CASCADE, blank=True, null=True)
```
Comment (Lassulus) [inline refugeerouter/models.py:44]: ah, missed that, refugees without a group doesn't make any sense. so we should always enforce one

Comment (0mbi) [reply refugeerouter/models.py:44]: My thought was: what if a refugee is moved out of a group. This one should not have a group.

**Pull request merged**